01/GlobalVariableExample.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+
+int x = 100;
+
+void IncreaseNumber()
+{
+	int y = 200;
+	y += 100;
+	x += 100;
+}
+
+int main()
+{
+	printf("%d\n", x);
+	IncreaseNumber();
+	printf("%d\n", x);
+	printf("%d\n", y);

```

Comment:
comment this out. We will say in the code slides that if you comment this out, it will result in compile error